src/ngx_http_lua_kong_apple_m1.c

```diff
@@ -20,11 +20,6 @@
 
 // macOS with M1 fixes, see: https://github.com/LuaJIT/LuaJIT/issues/205
```

Comment:
Also should remove this or change the style to C89.

Reply:
thanks